models/apple_store__device_report.sql

```diff
@@ -0,0 +1,106 @@
+with app_store_device_report as (
+
+    select *
+    from {{ var('app_store_device_report') }}
+)
+, downloads_device_report as (
+
+    select *
+    from {{ var('downloads_device_report') }}
+)
+
+, usage_device_report as (
+
+    select *
+    from {{ var('usage_device_report') }}
+)
+
+, crashes_device_report as (
```

Comment:
Is this correct formatting? I feel like this looks weird. Can we do the below instead?
```suggestion
with app_store_device_report as (

    select *
    from {{ var('app_store_device_report') }}
), 

downloads_device_report as (

    select *
    from {{ var('downloads_device_report') }}
),

usage_device_report as (

    select *
    from {{ var('usage_device_report') }}
), 

crashes_device_report as (
```

If so, then can we update the rest of the models and our style guide as well. Open to your thoughts!

Reply:
Good catch, will incorporate to align with the style guide!